lib/src/tasks/coverage/api.dart

```diff
@@ -332,12 +332,8 @@ class CoverageTask extends Task {
     for (int i = 1; i < collections.length; i++) {
       if (!collections[i].existsSync()) continue;
       String coverage = collections[i].readAsStringSync();
-      if (coverage.isEmpty) {
-//        mergedJson['coverage'].addAll({});
-      } else {
-        Map coverageJson = JSON.decode(collections[i].readAsStringSync());
-        mergedJson['coverage'].addAll(coverageJson['coverage']);
-      }
+      Map coverageJson = JSON.decode(collections[i].readAsStringSync());
+      mergedJson['coverage'].addAll(coverageJson['coverage']);
```

Comment:
I'm not actually sure on that one.. I seem to remember that an empty coverage file may just mean that there was no coverage to be collected, which could be the case since we get several observatory ports but only one or a subset of them actually have coverage to be collected.